### PR TITLE
webgpu： Add new ops

### LINKED ISSUE #3405: webgpu: Add new op to support some classfication models and super-resolution models
Add some operations that are frequently used in classification models and super-resolution models

- batchToSpaceND
- softmax
- avgPool

To get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

#### Code to reproduce the bug / link to feature request
If you would like to get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

GitHub issues for this repository are tracked in the [tfjs union repository](https://github.com/tensorflow/tfjs/issues).

Please file your issue there, following the guidance in [that issue template](https://github.com/tensorflow/tfjs/blob/master/ISSUE_TEMPLATE.md).

### PR DESCRIPTION
**Pull request by NALLEIN**
Fix #3405.
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/3406)

### CODE REVIEW

**Comment by annxingyuan** [comment]
@NALLEIN oops looks like you may need to update some imports

**Comment by NALLEIN** [comment]
> @NALLEIN oops looks like you may need to update some imports

I've fixed the imports problem, PTAL.
